git-keeper.py

```diff
@@ -67,9 +69,8 @@ def git_clone_upload(s3_client, gpg, recipients,
             output=repo_gpg,
             armor=False,
             always_trust=True)
-    upload2s3(s3_client, repo_gpg, s3_bucket, subfolder,
-              date, os.path.basename(
-                  repo_url) + '.tar.gpg')
+    upload2s3(s3_client, repo_gpg, s3_bucket, subfolder, date,
```

Comment:
I would just call `s3_client.upload_file(repo_tar, git_keeper_bucket,
                          os.path.join(subfolder, date, object_name))` directly here, and remove the `uplosad2s3` function, as it doesn't add anything.